unstructured/file_utils/encoding.py

```diff
@@ -27,6 +27,10 @@
 ]
 
 
+def format_encoding_str(encoding: str) -> str:
+    return encoding.lower().replace("_", "-")
```

Comment:
Could you add a quick comment here showing what the encoding looks like before and after?

Reply:
@MthwRobinson @cragwolfe Added comments showing what the encoding looks like before and after.